test/publish.test.js

```diff
@@ -1,11 +1,26 @@
 import test from 'ava';
 import nock from 'nock';
+import tempy from 'tempy';
 import {stub} from 'sinon';
 import publish from '../lib/publish';
 import authenticate from './helpers/mock-gitlab';
 
 /* eslint camelcase: ["error", {properties: "never"}] */
 
+const owner = 'test_user';
```

Comment:
See https://github.com/semantic-release/gitlab/pull/55#discussion_r236032266

All those variable should be in each test rather than set globally for the file.
I thought you made that change already. It might have been reverted somehow.

Reply:
done https://github.com/semantic-release/gitlab/pull/55/commits/5f340eaf26cef4760d96fb732464a1243dd68144 https://en.wikipedia.org/wiki/Don%27t_repeat_yourself